client/state/sites/selectors.js

```diff
@@ -145,7 +170,7 @@ export function isJetpackMinimumVersion( state, siteId, version ) {
  * @return {?String}        Site slug
  */
 export function getSiteSlug( state, siteId ) {
-	const site = getSite( state, siteId );
+	const site = getRawSite( state, siteId );
```

Comment:
The code below that checks site.options should be made redundant now and getSiteSLug should just return the `slug` attribute of `getSite`.